### PR TITLE
PMM-6854 Return status changing for PXC if status is empty

### PR DESCRIPTION
**Pull request by percona-csalguero**
When the XtraDB cluster is starting, Kubernetes returns an empty status that's replicated on the front end.
This tiket changes that behavior and returns status `Changing` if the status is empty.

|Component|Link|
|-----|-----|
|Jira|https://jira.percona.com/browse/PMM-6854|
|FB|https://github.com/Percona-Lab/pmm-submodules/pull/1285|

### CODE REVIEW

**Review by askomorokhov — APPROVED** [review]
Looks good to me.

**Review by BupycHuk — APPROVED** [review]

File: service/k8sclient/k8client.go [hunk 1]
```diff
@@ -349,6 +349,14 @@ func (c *K8Client) getPerconaXtraDBClusters(ctx context.Context) ([]XtraDBCluste
 	return res, nil
 }
 
+func getPXCState(state pxc.AppState) ClusterState {
+	clusterState, ok := pxcStatesMap[state]
+	if !ok {
```
Comment (BupycHuk) [inline service/k8sclient/k8client.go:354]: The problem with this approach that status in operators can be unknown.

**Comment by AlekSi** [comment]
@percona-csalguero Please resolve merge conflicts

**Review by AlekSi — COMMENTED** [review]
@percona-csalguero @BupycHuk As an idea: do we have an information about CR creation time? If yes, can we return Changing state for empty / unknown / unexpected status only if CR is less that, say, 10 seconds old?

I'm afraid about about the fact we hide unknown / unprotected statuses without even logging.

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**